docs/changes/2670.feature.rst

```diff
@@ -0,0 +1,10 @@
+Add new fields to the MuonParametersContainer and refactor the code using the existing containers.
```

Comment:
This reads like you only modified the container, but you also actually implemented the computation of these new features, right?

Maybe emphasize that a bit?